### ISSUE TITLE
SVM - error

### ISSUE THREAD

**Issue by rthapa26** [topepo/caret#1098]
Hi,

I am trying to use the SVM method. But got NAs while predicting from test data.  Is there any suggestions? I would appreciate it.

library(e1071)
library(kernlab)
model_svm<-train(nzv[index,],trainSet.y,method='svmRadial',trControl=fitControl,tuneLength=6)
testSet$model_svm <- predict(model_svm,testSet)
> testSet$model_svm <- predict(model_svm,testSet)
Warning message:
In method$predict(modelFit = modelFit, newdata = newdata, submodels = param) :
  kernlab class prediction calculations failed; returning NAs
> testSet$model_svm 
 [1] <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA>
[25] <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA> <NA>
Levels: N Y

**Comment by topepo** [comment]
> kernlab class prediction calculations failed

Since SVM models don't naturally compute probability estimates, a secondary model is computed by `kernlab::ksvm()` (called Platt scaling) that converts the decision values (similar to discriminant values) to probabilities. This is based on a sigmoidal model. If that model fails then the probability calculations fail. 

tl;dr It is an error in `kernlab::ksvm()` and there might not be anything that you can do about it.
